### PR TITLE
Update contributing

### PR DESCRIPTION
**Pull request by Mackiovello**
The instruction about PropertyMetadata was misinformation. 

The SignIn links require https://github.com/StarcounterApps/SignIn/pull/92

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,9 @@ Calendar
 
 View and edit **events** on a calendar. Handles multiple calendars. Supports the iCalendar format.
 
-Note: To Create Events you need to be signed in using [Sign In](https://github.com/Polyjuice/SignIn) or [Google Sign In](https://github.com/Polyjuice/GoogleSignIn).
+This app requires a user to be signed in to access certain pages, such as the settings page. This can be done with the [SignIn](https://github.com/starcounterapps/signin) app. If there are no users in the database, [create the first user](https://github.com/starcounterapps/signin#create-the-first-user). Otherwise, [sign in an existing user](https://github.com/starcounterapps/signin#sign-in)
 
-## Building
-
-To build, you need to obtain https://github.com/Smorgasbord-Development/PropertyMetadata
+For developer instructions, go to [CONTRIBUTING](CONTRIBUTING.md).
```
Comment (warpech) [inline README.md]: I propose this comes after a header:

```
## Developer instructions
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+2. Build the solution by executing `build.bat` in Windows or `./build.bat` in Git Bash
+3. Run it by executing `run.bat` in Windows or `./run.bat` in Git Bash
+4. Go to [http://localhost:8080/Calendar](http://localhost:8080/Calendar)
+
+### Build and run using Visual Studio
+
+1. Clone this repo to your local machine
+2. Open the `.sln` file in Visual Studio
+3. Build and run it using Debug in Visual Studio (**Debug** > **Start debugging** or <kbd>F5</kbd>)
+4. Go to [http://localhost:8080/Calendar](http://localhost:8080/Calendar)
```
Comment (warpech) [inline CONTRIBUTING.md]: Let's stick to lowercase URLs: http://localhost:8080/calendar

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Developer instructions
+
+## How to build and run
+
+### Build and run using CLI
+
+1. Clone this repo to your local machine
+2. Build the solution by executing `build.bat` in Windows or `./build.bat` in Git Bash
+3. Run it by executing `run.bat` in Windows or `./run.bat` in Git Bash
+4. Go to [http://localhost:8080/Calendar](http://localhost:8080/Calendar)
```
Comment (warpech) [inline CONTRIBUTING.md]: Let's stick to lowercase URLs: http://localhost:8080/calendar

**Comment by Mackiovello** [comment]
Thank you for the review! Fixed it according to your comments. Merging.

**Comment by Mackiovello** [comment]
Set the wrong branch to merge into. Noticed after merging from command line. Merged with this commit https://github.com/StarcounterApps/Calendar/commit/522bea300055fa8d1242be48b9a52bcb2088668d. Closing the PR

**Pull request closed without merging**